Source/WebKit/Shared/mac/WebGestureEvent.h

```diff
@@ -56,13 +56,18 @@ class WebGestureEvent : public WebEvent {
 
     float gestureScale() const { return m_gestureScale; }
     float gestureRotation() const { return m_gestureRotation; }
-    
+
+    void setCoalescedEvents(const Vector<WebGestureEvent>& coalescedEvents) { m_coalescedEvents = coalescedEvents; }
+    Vector<WebGestureEvent> coalescedEvents() const { return m_coalescedEvents; }
+
 private:
     bool isGestureEventType(WebEventType) const;
 
     WebCore::IntPoint m_position;
     float m_gestureScale;
     float m_gestureRotation;
+
+    Vector<WebGestureEvent> m_coalescedEvents;
```

Comment:
Is this member variable (along with the corresponding setter / getter) consulted anywhere?